vmngclient/api/task_status_api.py

```diff
@@ -56,7 +58,8 @@ def wait_for_completed(
         session (vManageSession): session
         action_id (str): inspected action id
         timeout_seconds (int): After this time, function will stop requesting action status
-        sleep_seconds (int): interval between action status requests
+        interval_seconds (int): interval between action status requests
+        delay_seconds (int): if Vmanage return empty list, after this time api call would be repeated
```

Comment:
```suggestion
        delay_seconds (int): if Vmanage didn't report task status, after this time api call would be repeated
```

Reply:
We already know that response could be empty list, but it should be more informative for the other users -